README.md

```diff
@@ -159,7 +159,7 @@ $ ./configure
 $ make && sudo make install
 ```
 
-When using the configure script, the generated `config.mk` file may override the
+When using the configure script, the generated `config.mk` the file may override the
```

Comment:
That's an extra `the` - it was correct before the change

Reply:
@jonas-schievink  Yes Correct. I have reverted it.